tests/e2e/test_utils.go

```diff
@@ -0,0 +1,711 @@
+package e2e
+
+import (
+	"context"
+	"crypto/md5"
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"mime"
+	"net/http"
+	"os"
+	"path/filepath"
+	"runtime"
+	"slices"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/sergi/go-diff/diffmatchpatch"
+	"github.com/stretchr/testify/assert"
+	"github.com/tebeka/selenium"
+	"github.com/tebeka/selenium/chrome"
+	"github.com/tebeka/selenium/firefox"
+)
+
+const FIREFOX string = "firefox"
+const CHROME string = "chrome"
+
+var currentPath string
+var DownloadPath string
+var ResourcesPath string
+var seleniumPort int
+var server *http.Server
+var MaxWaitTimeout time.Duration
+var rulesFile = "rules/har.yaml"
```

Comment:
This has to be dynamically constructed based on the input file extension.

Reply:
This will be handled in the future, once other file types sanitization is supported